app/models/patient.rb

```diff
@@ -78,8 +78,6 @@ def self.follow_ups_with(model_name, period, time_column: 'recorded_at', last: n
       .distinct
   end
 
-  private_class_method :follow_ups_with
```

Comment:
@kitallis Not sure the risk of making this public? I think its worth exposing it for now to silence the deprecation warning though.

Reply:
Nope, it's perfectly ok for this to be public, it's just private for clean-public-interface's sake.